src/components/ButtonSubmit.js

```diff
@@ -1,8 +1,17 @@
 import React from 'react'
+import { View } from 'react-native'
 import { Button } from '@rneui/base'
 
 import { styles } from './ButtonSubmit.styles'
 
+export const ButtonSubmitSticky = (props) => {
```

Comment:
personal-pref: Should we put it into its own file? it makes it a tiny little bit easier to spot I think :)

Reply:
@khamui I kind of thought of it like a container and we use this structure elsewhere like list and form fields, but maybe I should rename to include container?

Reply:
but I also don't really mind, let me know what you prefer and i'll change it :)

Reply:
actually I like the `..Sticky` as it makes it very clear what it is. If we never use the `ButtonSubmit` standalone, we can keep it like that and just rename the file to `ButtonSubmitSticky.js`.If we do use it standalone, I would separate it so we have the variant `ButtonSubmit` and `ButtonSubmitSticky`. Does that make sense?